my-app/src/HW37/hw-37.tsx

```diff
@@ -0,0 +1,28 @@
+import { Button } from "./buttonBurger"
```

Comment:
all components files names begin with a capital letter (it's a convention, not a rule)
ButtonBurger.tsx
Hw-37.tsx